### PR TITLE
feat(plugin-sonarqube): add links to each rating

### PR DESCRIPTION
**Pull request by jeskosda**
## Hey, I just made a Pull Request!

I think it's pretty handy to have individual links to the ratings as you are used to in the Sonarclound Overview. 

Oct-30-2020 16-05-13

### CODE REVIEW

File: plugins/sonarqube/src/api/index.ts [hunk 1]
```diff
@@ -105,6 +107,8 @@ export class SonarQubeApi {
       lastAnalysis: component.component.analysisDate,
       metrics,
       projectUrl: `${this.baseUrl}dashboard?id=${componentKey}`,
+      issuesUrl: `${this.baseUrl}project/issues?id=${componentKey}`,
+      componentMeasuresUrl: `${this.baseUrl}component_measures?id=${componentKey}`,
```
Comment (dhenneke) [inline plugins/sonarqube/src/api/index.ts:111]: How about something like this. Saves you an extra function in the `SonarQubeCard` and will be reusable in other cards easier.

```suggestion
      getIssueUrl: (type: string) => `${this.baseUrl}project/issues?id=${componentKey}&types=${type.toUpperCase()}&resolved=false`,
      getComponentMeasurementUrl: (metric: string) => `${this.baseUrl}component_measures?id=${componentKey}&metric=${metric.toLowerCase()}&resolved=false&view=list`,
    };
```

File: plugins/sonarqube/src/components/SonarQubeCard/RatingCard.tsx [hunk 2]
```diff
@@ -50,29 +50,33 @@ export const RatingCard = ({
...
+  link: string;
 }) => {
   const classes = useStyles();
 
   return (
-    <Grid item className={classes.root}>
-      <Grid item className={classes.upper}>
-        <Grid item className={classes.left}>
-          {leftSlot}
+    <a href={link}>
```
Comment (stefanalund) [inline plugins/sonarqube/src/components/SonarQubeCard/RatingCard.tsx:64]: Curious, what is the tradeoff between using `<a>` vs MUI's `<Link>` component?

Comment (jeskosda) [reply plugins/sonarqube/src/components/SonarQubeCard/RatingCard.tsx:64]: When using MUI Link you need to customize the color and the underline property:
`      <Link href={link} color="inherit" underline="none">
`
I could also replace it, if you prefer...

Comment (Rugvip) [reply plugins/sonarqube/src/components/SonarQubeCard/RatingCard.tsx:64]: Styling and a few extra utility props, but we should use it

Comment (stefanalund) [reply plugins/sonarqube/src/components/SonarQubeCard/RatingCard.tsx:64]: @jeskosda please update 👍

Comment (jeskosda) [reply plugins/sonarqube/src/components/SonarQubeCard/RatingCard.tsx]: Done 👍

**Review by stefanalund — APPROVED** [review]

**Review by dhenneke — APPROVED** [review]
🌟

**Review by Rugvip — APPROVED** [review]
Thanks! 👍

**Pull request merged**